couch2pg/src/importer.js

```diff
@@ -143,17 +144,16 @@ const importChangesBatch = async (couchDb, source) => {
   }
 
   const changes = await couchDb.changes({ limit: BATCH_SIZE, since: seq, seq_interval: BATCH_SIZE });
-  console.log('There are ' + changes.results.length + ' changes to process');
+  console.log(`There are ${changes.results.length} changes to process in ${dbName}`);
 
   const docsToDelete = [];
   const docsToDownload = [];
   changes.results.forEach(change => change.deleted ? docsToDelete.push(change) : docsToDownload.push(change));
 
-  console.info('There are ' +
-            docsToDelete.length + ' deletions and ' +
-            docsToDownload.length + ' new / changed documents');
+  console.info(`There are ${docsToDelete.length} deletions and ${docsToDownload.length} new changed documents ` +
```

Comment:
I believe there was a / between new and changed, to replace an "or". Can you please add the or? `new changed` documents is confusing. Maybe "new or edited documents"?